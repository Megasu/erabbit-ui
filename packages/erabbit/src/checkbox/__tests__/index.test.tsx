import { Checkbox } from '..'
import { mount } from '@vue/test-utils'

test('should render default Checkbox', () => {
  const wrapper = mount(Checkbox, {
    slots: {
      default: 'Checkbox'
    }
  })
  expect(wrapper.html()).toMatchSnapshot()
})
