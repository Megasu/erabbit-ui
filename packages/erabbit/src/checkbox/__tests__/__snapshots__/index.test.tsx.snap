// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render default button 1`] = `"<label class=\\"er-checkbox\\"><span class=\\"er-checkbox__input\\"><input class=\\"er-checkbox__origin\\" type=\\"checkbox\\"><span class=\\"er-checkbox__inner\\"></span></span><span class=\\"er-checkbox__label\\">Submit</span></label>"`;
